code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -264,6 +264,13 @@
 	list_reagents = list(/datum/reagent/consumable/ethanol/beer = 30)
 	foodtype = GRAIN | ALCOHOL
 
+/obj/item/reagent_containers/food/drinks/ftliver
```

Comment:
Please move all of these to files within the NSV13 folder for better compatibility.

Reply:
Two questions for you: Do I simply create a copy of the exact file path inside the NSV13 folder (eg nsv13/modules/food_and_drinks/drinks/drinks.dm, which doesn't exist currently) with the added code and nothing else in it?

And should I remake this PR from a clean branch to ensure that my commits have not touched the upstream files?

Reply:
Moving them out should *probably* be fine, and yes, make a path-identical blank file with your new code in it.